server/models/Profile.js

```diff
@@ -0,0 +1,51 @@
+const mongoose = require("mongoose");
+
+const profileSchema = new mongoose.Schema({
+  email: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  firstName: {
+    type: String,
+    required: true,
+  },
+  lastName: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+  },
+  city: {
+    type: String,
+    default: "",
+  },
+  phone: {
+    type: Number,
+  },
+  address: {
+    type: String,
+  },
+  photo: {
+    type: String,
+  },
+  availability: {
+    type: [String],
+    default: [],
+  },
+  booking_date: {
```

Comment:
We will eventually have a seperate model to keep track of user bookings, we can remove this field from the Profile model :)